Tema_1.ts

```diff
@@ -0,0 +1,100 @@
+class Todo {
+
+    name:string;
+    date:Date;
+    finished: boolean;
+    description: string;
+
+    constructor (name:string, date:Date, description:string) {
+        this.name=name;
+        this.date=date;
+        this.finished=false;
+        this.description=description;
+        
+    }
+
+     finish() {
+            if(this.finished==false) {
+                this.finished=true;
+            }
+            else{
+                console.log("Task finalizat!");
+            }
+    }
+
+    modify(target:string, modified:any) {
+        switch(target) {
+            case "name":
+                this.name=modified;
+                break;
+            case "date":
+                this.date=modified;
+                break;
+            case "description":
+                this.description=modified;
+                break;
+        }
+
+    }
+
+}
+
+
+
+class Todos {
+
+    taskList: Todo[];
+
+    constructor (){
+        this.taskList=[];
```

Comment:
Aici ai putea sa pui un parametru optional in constructor, poate vrei sa initializezi cu o lista, fara sa mai folosesti metoda add. Ai putea sa folosesti default parameter.
```
constructor(list: Todo[] = []) {
   this.taskList = list;
} 
```